AppCenterCrashes/AppCenterCrashes/Internals/Util/MSACErrorLogFormatter.m

```diff
@@ -315,14 +312,8 @@ + (MSACErrorReport *)errorReportFromLog:(MSACAppleErrorLog *)errorLog {
   NSString *exceptionName = errorLog.exceptionType;
   NSDate *appStartTime = errorLog.appLaunchTimestamp;
   NSDate *appErrorTime = errorLog.timestamp;
-  NSString *codeType = unknownString;
-  NSString *archName = unknownString;
-  if (errorLog.isKnownEncodingType && errorLog.primaryArchitectureId != nil) {
-    codeType = [self convertCodeTypeToString:errorLog.primaryArchitectureId.longValue];
-    if (errorLog.architectureVariantId != nil) {
-      archName = [self convertArchNameToString:errorLog.primaryArchitectureId.longValue subtype:errorLog.architectureVariantId.intValue];
-    }
-  }
+  NSString *codeType = [self convertCodeTypeToString:errorLog.primaryArchitectureId.longValue];
```

Comment:
Why don't we check `if (primaryArchitectureId != nil)` and `if (errorLog.architectureVariantId != nil)` anymore?
Also we are not setting `codeType = unknownString` anymore. Is it expected?

Reply:
rolled back this check

Reply:
It WAS expected. It's handled inside `convertCodeTypeToString` and `convertArchNameToString:errorLog` functions